src/flutter/code-tools.ts

```diff
@@ -206,12 +206,20 @@ export async function patchMain(
       )} to record user interactions and debug issues?`,
       enabledHint: 'recommended',
     });
+    features.push({
+      id: 'logs',
+      prompt: `Do you want to enable ${chalk.bold(
+        'Structured Logs',
+      )} to capture log messages with structured data?`,
+      enabledHint: 'optional',
+    });
```

Comment:
is this the same phrasing that is used by the other SDK wizards? just wanna make sure we align

Reply:
good catch, let me check

Reply:
updated to match others